File: Javascript/grade3.js
// 2739번
const fs = require('fs');
const a = Number(fs.readFileSync('/dev/stdin'));

for ( i = 1; i <= 9; i++ ) {
    console.log(`${a} * ${i} = ${a * i}`);
}

//10950번
const fs = require('fs');
const a = fs.readFileSync('/dev/stdin').toString().split('\n');

const len = Number(a.shift());
for ( i = 0; i < len; i++) {
    const arr = a[i].split(' ');
    console.log((Number(arr[0]) + Number(arr[1])));
};

//8393번
const fs = require('fs');
const n = Number(fs.readFileSync('/dev/stdin'));
const arr = [];
const reducer = (a, b) => a+b;

for ( i = 1; i <= n; i++ ) {
    arr.push(i);
}
console.log(arr.reduce(reducer));

// 2741번
const fs = require('fs');
const n = Number(fs.readFileSync('/dev/stdin'));
for (i = 1; i <= n; i++) {
    console.log(i);
}

// 2742번
const fs = require('fs');
const n = Number(fs.readFileSync('/dev/stdin'));
for (i = n; i >= 1; i--) {
    console.log(i);
}

// 11021번
const fs = require('fs');
const a = fs.readFileSync('/dev/stdin').toString().split('\n');
const len = a.shift();
for ( i = 0; i < len; i++) {
    const n = a[i].split(' ');
    console.log(`Case #${i+1}: ${Number(n[0])+Number(n[1])}`);
}

//11022번
const fs = require('fs');
const a = fs.readFileSync('/dev/stdin').toString().split('\n');
const len = a.shift();
for ( i = 0; i < len; i++) {
    const n = a[i].split(' ');
    console.log(`Case #${i+1}: ${n[0]} + ${n[1]} = ${Number(n[0])+Number(n[1])}`);
}

// 2438번
const fs = require('fs');
const n = Number(fs.readFileSync('/dev/stdin'));
const arr = [];
for (i = 1; i <= n; i++) {
    arr.push("*");
    console.log(arr.join(""));
}

// 2439번
const fs = require('fs');
const n = Number(fs.readFileSync('/dev/stdin'));
let star = "";
let space = "";
let space_arr = [];
for ( i = 0; i <= n-1; i++) {
    space_arr.push(space)
    space += " "
}
for ( i = n-1; i >= 0; i--) {
    star += "*";
    console.log(space_arr[i] + star);
}

// 10871번
const fs = require('fs');
const a = fs.readFileSync('/dev/stdin').toString().split('\n');
const len = Number(a[0].toString().split(' ')[0]);
const n = Number(a[0].toString().split(' ')[1]);
const arr = a[1].split(' ');
const answer = [];
for ( i = 0; i < arr.length; i++ ) {
    arr[i] = Number(arr[i]);
}
for ( i = 0; i < len; i++ ) {
    if ( arr[i] < n ) {
        answer.push(arr[i]);
    }
}
console.log(answer.join(" "));

File: Python/grade3.py
# 2739번
a = int(input(""))
for i in range(1, 10):
    print(a, '*', i, '=', a * i)

# 10950번
len = int(input(""))
for i in range(len):
    a, b = map(int, input("").split(" "))
    print(a+b)
    
# 8393번
n = int(input(""))
list = []
for i in range(1, n+1):
    list.append(i)
print(sum(list))

# 15552번
import sys
len = int(input())
for i in range(len):
    a, b = map(int, sys.stdin.readline().split())
    print(a + b)

# 2741번
n = int(input())
for i in range(1, n+1):
    print(i)

# 2742번
n = int(input())
for i in range(n, 0, -1):
    print(i)

# 11021번
import sys
len = int(input())
for i in range(1, len+1):
    a, b = map(int, sys.stdin.readline().split())
    print("Case #{}: {}".format(i, a+b))

# 11022번
import sys
len = int(input())
for i in range(1, len+1):
    a, b = map(int, sys.stdin.readline().split())
    print("Case #{}: {} + {} = {}".format(i, a, b, a+b))

# 2438번
len = int(input())
for i in range(1, len+1):
    print("*" * i)

# 2439번
len = int(input())
for i in range(1, len+1):
    print(' '*(len - i) + '*'*i)

# 10871번
a = list(map(int, input().split()))
arr = list(map(int, input().split()))
answer = []
for ele in arr:
    if a[1] > ele:
        answer.append(str(ele))
print(" ".join(answer))